routes/api.php

```diff
@@ -147,6 +147,8 @@
     'only' => ['update', 'destroy'],
 ]);
 
+$router->put('notifications', 'NotificationController@updateAll');
```

Comment:
I think I would prefer `notifications/all` as URL. This way there is less chance to update all notifications by accident if the ID is forgotten (e.g. in a custom script). While it is very unlikely that people use custom scripts to update notifications, it's a more robust default.